filecoin-proofs/src/api/post.rs

```diff
@@ -662,7 +662,7 @@ pub fn verify_winning_post(
         &pub_inputs,
         &proof,
         &fallback::ChallengeRequirements {
-            challenge_count: param_challenge_count,
+            challenge_count: post_config.challenge_count * replicas.len(),
```

Comment:
I would slightly rather see this using `post_config.sector_count` than `replicas.len()` (although I understand you have asserted them to be equal above). The former expresses exactly what we mean to require here.

Reply:
fixed